client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -414,20 +430,115 @@ export class PluginsList extends Component {
 		}
 	}
 
+	bulkActionDialog = ( actionName, selectedPlugin ) => {
+		const { plugins, translate } = this.props;
+		const selectedPlugins = selectedPlugin ? [ selectedPlugin ] : plugins.filter( this.isSelected );
+		const pluginsCount = selectedPlugins.length;
+		const pluginsSize = pluginsCount === 1 ? 'plugin' : 'plugins';
+		const isJetpackIncluded = selectedPlugins.some( ( { slug } ) => slug === 'jetpack' );
+
+		switch ( actionName ) {
+			case 'activate':
+				acceptDialog(
+					<div>
+						<span>{ this.getConfirmationText( selectedPlugins, 'activate', 'on' ) }</span>
+					</div>,
+					( accepted ) => this.activateSelected( accepted ),
+					translate( 'Activate %(numberOfPlugins)d %(plugins)s', {
```

Comment:
I feel like translations like these might cause an issue.

We can use

```
const translationArgs = {
   args: { count }, 
   count,
};
```
and

```
translate(
    'Activate on %(count)s plugin',
    'Activate on %(count)s plugins', // plural form
    translationArgs
 )
```